.serverless/cloudformation-template-update-stack.json

```diff
@@ -167,7 +167,7 @@
           "S3Bucket": {
             "Ref": "ServerlessDeploymentBucket"
           },
-          "S3Key": "serverless/full-lambda-function/dev/1666098617776-2022-10-18T13:10:17.776Z/full-lambda-function.zip"
+          "S3Key": "serverless/full-lambda-function/dev/1666255668655-2022-10-20T08:47:48.655Z/full-lambda-function.zip"
```

Comment:
The entire .serverless folder should be deleted and part of gitignore then

Reply:
deleted now